site/app/entities/poll/Poll.php

```diff
@@ -79,6 +81,9 @@ public function __construct(string $name, string $question, string $question_typ
         $this->setReleaseHistogram($release_histogram);
         $this->setReleaseAnswer($release_answer);
         $this->setImagePath($image_path);
+        $this->setAllowsCustomOptions(false);
```

Comment:
I had to modify the way image_path was set passed to the constructor. Is there a better way to set defaults for parameters? [4eecc6d](https://github.com/Submitty/Submitty/pull/10197/commits/4eecc6d3d76096667408c8f239158f27e7f4c2a3)

Reply:
I'm a bit confused about what the question is.  Shouldn't the custom options field be able to be set when constructing a Poll object rather than using a setter immediately afterwards all the time?